src/app/components/sign-up-modal/components/dynamic-form-field/dynamic-form-field.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input, forwardRef } from '@angular/core';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms';
+import { FormField } from '../../form-field.model';
+
+@Component({
+  selector: 'app-dynamic-form-field[field][name]',
+  templateUrl: './dynamic-form-field.component.html',
+  styleUrls: ['./dynamic-form-field.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => DynamicFormFieldComponent),
+      multi: true,
+    },
+  ],
+})
+export class DynamicFormFieldComponent implements ControlValueAccessor {
+  @Input() name!: string;
+
+  @Input() value!: string | boolean;
+
+  @Input() field!: FormField;
+
+  private onChange: (value: any) => void = () => {};
+
+  private onTouched: () => void = () => {};
```

Comment:
Is it template for the function in the future?

Below you have  registerOnChange, registerOnTouched functions